datarepo-clienttests/src/main/java/runner/DisruptiveScript.java

```diff
@@ -0,0 +1,39 @@
+package runner;
+
+import java.util.List;
+import runner.config.TestUserSpecification;
+
+public abstract class DisruptiveScript {
```

Comment:
Nit: I think all the comments here need to be updated for this class. (e.g. setup/cleanup/userjourney methods don't exist).

Reply:
Ok, I think I've updated the descriptions.